test/dotnet-help.Tests/GivenThatIWantToShowHelpForDotnetCommand.cs

```diff
@@ -65,5 +66,33 @@ public void WhenHelpOptionIsPassedToDotnetItPrintsUsage(string helpArg)
             cmd.Should().Pass();
             cmd.StdOut.Should().ContainVisuallySameFragment(HelpText);
         }
+
+        [Theory]
+        [InlineData("pack", "Usage: dotnet pack [arguments] [options] [args]")]
+        [InlineData("new", "Usage: dotnet new [arguments] [options]")]
+        [InlineData("restore", "Usage: dotnet restore [arguments] [options] [args]")]
+        [InlineData("build", "Usage: dotnet build [arguments] [options] [args]")]
+        [InlineData("publish", "Usage: dotnet publish [arguments] [options] [args]")]
+        [InlineData("run", "Usage: dotnet run [options] [[--] <additional arguments>...]]")]
+        [InlineData("test", "Usage: dotnet test [arguments] [options] [args]")]
+        [InlineData("migrate", "Usage: dotnet migrate [arguments] [options]")]
+        [InlineData("clean", "Usage: dotnet clean [arguments] [options] [args]")]
+        [InlineData("sln", "Usage: dotnet sln [arguments] [options] [command]")]
+        [InlineData("sln add", "Usage: dotnet sln <SLN_FILE> add [options] [args]")]
+        [InlineData("sln list", "Usage: dotnet sln <SLN_FILE> list [options]")]
+        [InlineData("sln remove", "Usage: dotnet sln <SLN_FILE> remove [options] [args]")]
+        [InlineData("nuget", "Usage: dotnet nuget [options] [command]")]
+        // [InlineData("msbuild", "")]
+        // [InlineData("vstest", "")]
```

Comment:
Comment included as a placeholder for further discussion.

The output here is also inconsistent with the others. 

It doesn't understand the `-h` option: 

```
$ dotnet vstest -h
Microsoft (R) Test Execution Command Line Tool Version 15.0.0.0
Copyright (c) Microsoft Corporation.  All rights reserved.

Error: The test source file "/Users/jonsequitur/dev/cli/-h" provided was not found.
```

The `--help` option is closer to the others but still inconsistent:

```
$ dotnet vstest --help
Microsoft (R) Test Execution Command Line Tool Version 15.0.0.0
Copyright (c) Microsoft Corporation.  All rights reserved.

Usage: vstest.console.exe [Arguments] [Options] [[--] <args>...]]

Description: Runs tests from the specified files.
```